deploy/crds/infra.watch_v1beta1_servicetelemetry_cr.yaml

```diff
@@ -3,13 +3,26 @@ kind: ServiceTelemetry
 metadata:
   name: default
 spec:
+  alerting:
+    enabled: true
+    alertmanager:
+      storage:
+        strategy: ephemeral
```

Comment:
There is a discussion ongoing about what the default should be at (ephemeral vs persistent).

I am considering setting the undefined default to be `persistent` (as it was in 1.0.3) but to set the example CR (which is what you'd get if you didn't override anything when doing a `ServiceTelemetry` object creation from the UI where this CR is loaded) to `ephemeral` to make the user experience a bit nicer when doing a test deployment.

@csibbitt what do you think of that approach? I'm not sure if that actually satisfies the concern of having customers running ephemeral in production though?

Reply:
It seems rather strange to have one default behavior from the CLI and a different one if you click through the example defaults in the OLM GUI; so I'm not sure I can get behind that solution either. And yeah, it doesn't really address my concern. My personal opinion on this is two parts:

1. Having a default production setup that doesn't lose my data is more important than a default production setup that "just works" and does lose my data. 
2. If this default is meant to make things easier "in development" then I suggest the patch that I always apply after I `git pull` this project:

```
[csibbitt@laptop service-telemetry-operator (master) ]$ git diff
diff --git a/deploy/quickstart.sh b/deploy/quickstart.sh
index 3bafad7..6899f9a 100755
--- a/deploy/quickstart.sh
+++ b/deploy/quickstart.sh
@@ -5,4 +5,5 @@ oc new-project "${OCP_PROJECT}"
 ansible-playbook \
     --extra-vars namespace="${OCP_PROJECT}" \
     --extra-vars __local_build_enabled=false \
+    --extra-vars __service_telemetry_storage_ephemeral_enabled=true \
     ${REL}/../build/run-ci.yaml
```
 Ideally we'd have a PV based solution that "just works" for some prescribed platform (OCS for example?) and get the best of both worlds. In the meantime I prefer the safe default to the easy one.

Reply:
OK that works for me. I will change this in a subsequent PR.